src/screens/homePageAuth/HomeScreen.js

```diff
@@ -9,7 +9,7 @@ import {
     Dimensions
 } from "react-native";
 import HPANavigation from "../../constants/HPANavigation";
-import logoVertical from '../../../assets/logoVertical.png'
+import logoVertical from '../../../assets/appAssets/logoVertical.png'
```

Comment:
```suggestion
import logoVertical from '../../../assets/appAssets/logoVertical.png'
```

Por que `assets/appAssets` y no solo `assets/app`? ya "assets" le da el contexto de "appAssets"